resources/assets/coffee/react/beatmapset-page/main.coffee

```diff
@@ -90,6 +90,13 @@ export class Main extends React.Component
     else
       @state.enabledMods
 
+    @setState
+      enabledMods: enabledMods
```

Comment:
the check for not-supporter viewing not-global leaderboard (L80) needs to come after this, else the mods menu in that case looks accessible but does nothing on clicks

Reply:
Merge the conditional.